src/OpenTelemetry.Instrumentation.Http/HttpClientInstrumentationOptions.cs

```diff
@@ -44,32 +44,10 @@ public class HttpClientInstrumentationOptions
         /// </summary>
         public Func<HttpRequestMessage, bool> FilterFunc { get; set; }
 
-        internal static bool IsInternalUrl(Uri requestUri)
-        {
-            var originalString = requestUri.OriginalString;
-
-            // zipkin
-            if (originalString.Contains(":9411/api/v2/spans"))
-            {
-                return true;
-            }
-
-            return false;
-        }
-
         internal bool EventFilter(string activityName, object arg1)
         {
             if (TryParseHttpRequestMessage(activityName, arg1, out HttpRequestMessage requestMessage))
             {
-                Uri requestUri;
```

Comment:
@cijothomas @alanwest @eddynaka We could probably improve the perf on this now by changing the order of things.
```csharp
internal bool EventFilter(string activityName, object arg1)
{
   return (this.FilterFunc == null || !TryParseHttpRequestMessage(activityName, arg1, out HttpRequestMessage requestMessage))
      ? true
      : this.FilterFunc(requestMessage);
}
```

Maybe also add the inline hint on this & TryParseHttpRequestMessage?